File: 0344-reverse-string/0344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int low = 0,high = s.size()-1;
        
        while(low<high){
            int temp = s[low];
            s[low]=s[high];
            s[high]=temp;
            low++;
            high--;
        }
    }
};

File: 0344-reverse-string/0344-reverse-string.java
class Solution { // tc & sc is O(1)
    public void reverseString(char[] s) {
        int low = 0,high = s.length-1;
        while(low<high){
            char temp = s[low];
            s[low]=s[high];
            s[high]=temp;
            low++;
            high--;
        }
    }
}